create_dockerfile.sh

```diff
@@ -1,13 +1,16 @@
-docker run --rm kaczmarj/neurodocker:v0.3.2 generate -b neurodebian:stretch-non-free -p apt \
-    --install fsl \
+docker run --rm kaczmarj/neurodocker:0.4.3 generate docker -b neurodebian:stretch-non-free -p apt \
+    --install fsl gcc g++ software-properties-common \
     --user=neuro \
     --add-to-entrypoint "source /etc/fsl/fsl.sh" \
-    --miniconda env_name=nimare \
+    --miniconda create_env=nimare \
```

Comment:
```suggestion
     --copy . /opt/NiMARE/ \
    --miniconda create_env=nimare \
```

If we want to be similar to [fmriprep](https://github.com/poldracklab/fmriprep/blob/dd0f4d4dfc26b107dfd9073760c2d03940051552/Dockerfile#L170) we can build the container around the current code in the repository as opposed to the master branch. This will make it easier to run tests on circleci with the container, since the image will be built around the user's code.